app/controllers/hello_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HelloController < ApplicationController
```

Comment:
Let's give this controller a more general name like `HomeController` or `HomePageController` for example.